app/controllers/labware_controller.rb

```diff
@@ -31,7 +31,7 @@ def index # rubocop:todo Metrics/AbcSize, Metrics/MethodLength
   def show
     @source_plates = @asset.source_plates
     respond_to do |format|
-      format.html
+      format.html { @aliquots = @asset.aliquots.include_summary.paginate(page: params[:page], per_page: 384) }
```

Comment:
Maybe pull 384 out to a constant with a name suggesting why 384, or this whole block could be a in a private method possibly (same for the other controllers below)